com.unity.render-pipelines.universal/Runtime/Passes/DeferredPass.cs

```diff
@@ -28,6 +28,15 @@ public override void Configure(CommandBuffer cmd, RenderTextureDescriptor camera
         {
             RenderTargetIdentifier lightingAttachmentId = m_DeferredLights.GbufferAttachmentIdentifiers[m_DeferredLights.GBufferLightingIndex];
             RenderTargetIdentifier depthAttachmentId = m_DeferredLights.DepthAttachmentIdentifier;
+            if (m_DeferredLights.UseRenderPass)
```

Comment:
I still quite don't follow why we need this settting per-pass instead of checking the useRenderPass in global asset setting?

Reply:
By this setting per-pass you mean why DeferredLights has this variable? In this case deferred lights technically is not a Pass but a class that just does a lot of logic for GBuffer and Deferred passes. It actually copies the global asset setting and in this case it's safer here to check for the global setting (as the ScriptableRenderPass.useNativeRenderPass could be true even if falling back). I agree this is a bit confusing and messy and will try to make it more understandable